src/test/java/com/yahoo/bullet/bql/integration/SelectTest.java

```diff
@@ -152,4 +152,17 @@ public void testTooManyQueryTypes() {
         Assert.assertEquals(errors.get(0).getError(), "Query consists of multiple aggregation types.");
         Assert.assertEquals(errors.size(), 1);
     }
+
+    @Test
+    public void testSelectNow() {
+        long before = System.currentTimeMillis();
+        build("SELECT NOW FROM STREAM()");
```

Comment:
How does this clash with columns with that name in the schema? Does it prefer the literal or the schema column? I would guess the literal since it's a parsing thing. Anyway to avoid this if we have a schema column with that name?

Reply:
Double quotes as always